File: baekjoon/4153.cpp
#define _CRT_SECURE_NO_WARNINGS

#include <bits/stdc++.h>

using namespace std;

int main(void) {
	ios_base::sync_with_stdio(0);
	cin.tie(0);

	int a, b, c, temp;
	while (1) {
		cin >> a >> b >> c;
		if (a == 0)
			break;

		if (a > c) {
			temp = c;
			c = a;
			a = temp;
		}
		if (b > c) {
			temp = c;
			c = b;
			b = temp;
		}
		
		if (c * c == b * b + a * a)
			cout << "right\n";
		else
			cout << "wrong\n";
	}

	return 0;
}

File: baekjoon/4153.py
import sys

while True:
    a, b, c = map(int, sys.stdin.readline().split())
    if a == b == c == 0:
        break
    a *= a
    b *= b
    c *= c
    if (a+b == c) or (a+c == b) or (b+c == a):
        print("right")
    else:
        print("wrong")